db/migrations/20230207110032_create_products_table.sql

```diff
@@ -0,0 +1,16 @@
+-- migrate:up
+CREATE TABLE products (
+  id int PRIMARY KEY NOT NULL AUTO_INCREMENT,
+  name varchar(255) NOT NULL,
+  description varchar(2700),
+  subcategory_id int NOT NULL,
+  price decimal,
+  discount_price decimal,
+  product_img varchar(2700),
+  is_present boolean,
```

Comment:
present gift 둘다 동일한 의미를 가지지만 is_gift로 수정해주면 좋을 것 같습니다~!
```suggestion
  is_gift boolean,
```